docs/local-testing-w-notifications.md

```diff
@@ -1,45 +1,96 @@
 # Running Notifications + Relations + Inventory using Local Built Binaries
 
+This process goes through running Notifications, Inventory API and Relations API locally using built binaries. Since there are database dependencies involved, podman/docker are leveraged to handle running databases.
+
+## Prerequisites
+
+You'll need to following tools:
```

Comment:
```suggestion
You'll need the following tools:
```

Reply:
As you can see, my copy+paste game is much better than my spelling/grammar checking skills 😂